eng/scripts/Update-PkgVersion.ps1

```diff
@@ -0,0 +1,32 @@
+
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $PackageName,
+  [string] $NewVersionString,
+  [string] $ReleaseDate
+)
+
+. (Join-Path $PSScriptRoot ".." common scripts common.ps1)
+
+$pkgProperties = Get-PkgProperties -PackageName $PackageName
+$packageVersion = $pkgProperties.Version
+
+$packageSemVer = [AzureEngSemanticVersion]::new($packageVersion)
+
+if ([System.String]::IsNullOrEmpty($NewVersionString))
+{
+    $packageSemVer.IncrementAndSetToPrerelease()
+    & "${EngCommonScriptsDir}/Update-ChangeLog.ps1" -Version $packageSemVer.ToString() `
+    -ChangelogPath $pkgProperties.ChangeLogPath -Unreleased $True
+}
+else
+{
+    $packageSemVer = [AzureEngSemanticVersion]::new($NewVersionString)
+    & "${EngCommonScriptsDir}/Update-ChangeLog.ps1" -Version $packageSemVer.ToString() `
+    -ChangelogPath $pkgProperties.ChangeLogPath -Unreleased $False `
+    -ReplaceLatestEntryTitle $True -ReleaseDate $ReleaseDate
+}
+
+$versionFileContent = Get-Content -Path $pkgProperties.versionFile -Raw
+Set-Content -Path $pkgProperties.versionFile ($versionFileContent -replace $packageVersion, $packageSemVer.ToString())
```

Comment:
Do we think this replace might accidently replace more then the one version? Or do we expect there to only ever be the one version in the version file?

Reply:
I updted it to replace on the specific line.